### DISCUSSION TITLE
Connecting to wallet problem

### DISCUSSION THREAD

**Discussion by Litvinov999** [Chia-Network/chia-blockchain#8787] [Support]
I have some problem with connecting to my wallet after power failure, pls help me to fix this. Try to uninstall prgram and try to install other version didn`t help.
![изображение_2021-10-10_134708](https://user-images.githubusercontent.com/92252805/136692399-403844cd-120f-4b8b-87bc-63b8e32a3df3.png)

**Comment by BrandtH22** [comment]
@Litvinov999 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - this issue is likely a corrupt wallet database, you can stop chia, delete the wallet db here `~\.chia\mainnet\wallet\db`, and restart chia to get it to start syncing again.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
